arbeitszeit_flask/url_index.py

```diff
@@ -42,6 +43,40 @@ def get_confirmation_url(self, token: str) -> str:
             endpoint="auth.confirm_email_member", token=token, _external=True
         )
 
+    def get_global_barplot_for_certificates_url(
```

Comment:
I looks to me like the implementations for the MemberUrlIndex and the CompanyUrlIndex are identical. I would suggest to implement one single PlotUrlIndex to reduce code duplication.